src/components/LabelMenus/index.js

```diff
@@ -1,70 +1,81 @@
 import styles from "./styles.module.css"
 import React, { Component } from "react"
-import LabelTitle from "../LabelTitle/index"
-import SectionBase from "../SectionBase/index.js"
-
+import { connect } from "react-redux"
+import { sendBaseInfo } from "../../store/bases"
 class LabelMenus extends Component {
   constructor(props) {
     super(props)
     this.state = {
       showHighlight: this.props.highlightValue,
-      titleList: ["Use Cases and Alerts", "Overview", "Maintenance"],
+      menus: [
+        {
+          title: "Overview",
+          desc:
+            "Overview information about the dataset including Description, Provenance, Composition, and Collection.",
+        },
+        {
+          title: "Use Cases and Alerts",
+          desc:
+            "Relevant alerts for data practitioners who intent to use this dataset for specific use cases (types of analyses).",
+        },
+        {
+          title: "Maintenance",
+          desc:
+            "Information about the ongoing management of the dataset, such as how the data will be maintained, updated, and the best contact for further inquiries.",
+        },
```

Comment:
Since the info in here is not meant to change, I think it might be best as a `const` declared above the component. That way we don't run the risk of it inadvertently changes some time down the road when declared as a state.

Reply:
Do you mean `<menusLabel: {
menus: [
....
]
},`

because I can't place const outside of render.

Reply:
I think you declare it outside of the component entirely, below the imports but above the component declaration, it could work!